.release/ci.hcl

```diff
@@ -18,159 +18,31 @@ project "nomad" {
   }
 }
 
-event "merge" {
-  // "entrypoint" to use if build is not run automatically  // i.e. send "merge" complete signal to orchestrator to trigger build
-}
-
 event "build" {
-  depends = ["merge"]
-
   action "build" {
     organization = "hashicorp"
     repository   = "nomad"
     workflow     = "build"
   }
 }
 
-event "upload-dev" {
+event "prepare" {
   depends = ["build"]
 
-  action "upload-dev" {
-    organization = "hashicorp"
-    repository   = "crt-workflows-common"
-    workflow     = "upload-dev"
-  }
-
-  notification {
-    on = "fail"
-  }
-}
-
-event "security-scan-binaries" {
-  depends = ["upload-dev"]
-
-  action "security-scan-binaries" {
-    organization = "hashicorp"
-    repository   = "crt-workflows-common"
-    workflow     = "security-scan-binaries"
-    config       = "security-scan.hcl"
-  }
-
-  notification {
-    on = "fail"
-  }
-}
-
-event "notarize-darwin-amd64" {
-  depends = ["security-scan-binaries"]
-
-  action "notarize-darwin-amd64" {
+  action "prepare" {
     organization = "hashicorp"
     repository   = "crt-workflows-common"
-    workflow     = "notarize-darwin-amd64"
+    workflow     = "prepare"
+    depends      = ["build"]
```

Comment:
Does this belong in the action stanza?  The event already declares this dependency, and I don't see any other examples of `depends` within `action`.

Reply:
Yeah, this is the way it's been configured in all other projects: https://github.com/hashicorp/crt-core-helloworld/blob/main/.release/ci.hcl#L45, https://github.com/hashicorp/tfc-agent/blob/main/.release/ci.hcl#L34, https://github.com/hashicorp/consul/blob/main/.release/ci.hcl#L33. I can't tell you the exact reason for this, though :-D Scott would likely know!